ERROR - 2025-01-17 06:39:53 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:39:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:53 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:39:55 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:39:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:55 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:39:57 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:39:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:57 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:39:59 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:39:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:39:59 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:40:03 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:40:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:03 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:40:06 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:40:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:06 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 06:40:07 --> mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause' in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /home/customer/www/showscreen.app/public_html/app/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
#7 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
#8 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-01-17 06:40:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
[Method: GET, Route: ausentes]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 2 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 4 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 06:40:07 --> [Caused by] mysqli_sql_exception: Unknown column 'user_ficha' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `user_ficha` = \'1\'
AND `user_activo` = \'1\'
ORDER BY `nombre_usuario` ASC')
 4 SYSTEMPATH/Database/BaseBuilder.php(1636): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_ficha` = :user_ficha:
AND `user_activo` = :user_activo:
ORDER BY `nombre_usuario` ASC', [...], false)
 5 SYSTEMPATH/Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH/Controllers/Fichar.php(327): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Ausentes()
 9 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
10 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-17 12:12:06 --> mysqli_sql_exception: Deadlock found when trying to get lock; try restarting transaction in /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `proceso...', 0)
#1 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `proceso...')
#2 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `proceso...')
#3 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `proceso...', Array, false)
#4 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/BaseModel.php(1028): CodeIgniter\Model->doUpdate(NULL, Array)
#6 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Model.php(865): CodeIgniter\BaseModel->update(NULL, Array)
#7 /home/customer/www/showscreen.app/public_html/app/Controllers/Procesos_pedidos.php(168): CodeIgniter\Model->update()
#8 /home/customer/www/showscreen.app/public_html/app/Controllers/Procesos_pedidos.php(126): App\Controllers\Procesos_pedidos->calcularYActualizarOrdenParaMaquina('5', '40', '44568', 2)
#9 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(933): App\Controllers\Procesos_pedidos->actualizarEstadoProcesos()
#10 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Procesos_pedidos))
#11 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 /home/customer/www/showscreen.app/public_html/vendor/codeigniter4/codeigniter4/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /home/customer/www/showscreen.app/public_html/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-01-17 12:12:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Deadlock found when trying to get lock; try restarting transaction
[Method: POST, Route: procesos_pedidos/actualizarEstadoProcesos]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `procesos_pedidos` SET `estado` = :estado.1:, `id_maquina` = :id_maquina:, `orden` = :orden:
WHERE `id_proceso` = :id_proceso:
AND `id_linea_pedido` = :id_linea_pedido:
AND `estado` = :estado:', [...], false)
 2 SYSTEMPATH/Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH/BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 4 SYSTEMPATH/Model.php(865): CodeIgniter\BaseModel->update(null, [...])
 5 APPPATH/Controllers/Procesos_pedidos.php(168): CodeIgniter\Model->update()
 6 APPPATH/Controllers/Procesos_pedidos.php(126): App\Controllers\Procesos_pedidos->calcularYActualizarOrdenParaMaquina('5', '40', '44568', 2)
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Procesos_pedidos->actualizarEstadoProcesos()
 8 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Procesos_pedidos))
 9 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 12:12:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Deadlock found when trying to get lock; try restarting transaction
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `procesos_pedidos` SET `estado` = 3, `id_maquina` = \'5\', `orden` = 2
WHERE `id_proceso` = \'40\'
AND `id_linea_pedido` = \'44568\'
AND `estado` = 2')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `procesos_pedidos` SET `estado` = 3, `id_maquina` = \'5\', `orden` = 2
WHERE `id_proceso` = \'40\'
AND `id_linea_pedido` = \'44568\'
AND `estado` = 2')
 3 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `procesos_pedidos` SET `estado` = :estado.1:, `id_maquina` = :id_maquina:, `orden` = :orden:
WHERE `id_proceso` = :id_proceso:
AND `id_linea_pedido` = :id_linea_pedido:
AND `estado` = :estado:', [...], false)
 4 SYSTEMPATH/Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH/BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 6 SYSTEMPATH/Model.php(865): CodeIgniter\BaseModel->update(null, [...])
 7 APPPATH/Controllers/Procesos_pedidos.php(168): CodeIgniter\Model->update()
 8 APPPATH/Controllers/Procesos_pedidos.php(126): App\Controllers\Procesos_pedidos->calcularYActualizarOrdenParaMaquina('5', '40', '44568', 2)
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Procesos_pedidos->actualizarEstadoProcesos()
10 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Procesos_pedidos))
11 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 12:12:06 --> [Caused by] mysqli_sql_exception: Deadlock found when trying to get lock; try restarting transaction
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `procesos_pedidos` SET `estado` = 3, `id_maquina` = \'5\', `orden` = 2
WHERE `id_proceso` = \'40\'
AND `id_linea_pedido` = \'44568\'
AND `estado` = 2', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `procesos_pedidos` SET `estado` = 3, `id_maquina` = \'5\', `orden` = 2
WHERE `id_proceso` = \'40\'
AND `id_linea_pedido` = \'44568\'
AND `estado` = 2')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `procesos_pedidos` SET `estado` = 3, `id_maquina` = \'5\', `orden` = 2
WHERE `id_proceso` = \'40\'
AND `id_linea_pedido` = \'44568\'
AND `estado` = 2')
 4 SYSTEMPATH/Database/BaseBuilder.php(2508): CodeIgniter\Database\BaseConnection->query('UPDATE `procesos_pedidos` SET `estado` = :estado.1:, `id_maquina` = :id_maquina:, `orden` = :orden:
WHERE `id_proceso` = :id_proceso:
AND `id_linea_pedido` = :id_linea_pedido:
AND `estado` = :estado:', [...], false)
 5 SYSTEMPATH/Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH/BaseModel.php(1028): CodeIgniter\Model->doUpdate(null, [...])
 7 SYSTEMPATH/Model.php(865): CodeIgniter\BaseModel->update(null, [...])
 8 APPPATH/Controllers/Procesos_pedidos.php(168): CodeIgniter\Model->update()
 9 APPPATH/Controllers/Procesos_pedidos.php(126): App\Controllers\Procesos_pedidos->calcularYActualizarOrdenParaMaquina('5', '40', '44568', 2)
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Procesos_pedidos->actualizarEstadoProcesos()
11 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Procesos_pedidos))
12 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-17 15:00:10 --> ErrorException: Trying to access array offset on value of type null
[Method: GET, Route: salir/22]
in APPPATH/Views/salir.php on line 19.
 1 APPPATH/Views/salir.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', '/home/customer/www/showscreen.app/public_html/app/Views/salir.php', 19)
 2 SYSTEMPATH/View/View.php(234): include('/home/customer/www/showscreen.app/public_html/app/Views/salir.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1171): CodeIgniter\View\View->render('salir', [], true)
 5 APPPATH/Controllers/Fichar.php(404): view('salir', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Fichar->Salir('22')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Fichar))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(29): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
